core/src/main/java/com/linecorp/armeria/common/logging/DefaultLogWriter.java

```diff
@@ -117,6 +117,11 @@ public void logResponse(RequestLog log) {
         }
     }
 
+    @Override
+    public void log(RequestLog log) {
+        // noop;
```

Comment:
Perhaps we should implement this for completeness, although it's not gonna be used in many cases?

Reply:
Should I control the logging output using `loggingAtOnce` like `KafkaLogWriter`?